content/08-cardano-sidechains/03-example-evm-sidechain/02-create-and-fund-accounts.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Create and fund accounts
+metaTitle: Create and fund accounts
+---
+
+### Prerequisites:
+
+1. Ability to make an HTTP request.
+2. Optional: Download `evm-sc-cli` binary and add to your path: https://github.com/input-output-hk/sidechains-tooling
+
+# How to fund your mainchain account with test ADA tokens
+
+In this tutorial, you will learn how to claim test Ada tokens on the mainchain testnet.
+
+## Generating keys
+
+To generate keys for the cardano testnet you need the `cardano-cli` [binary](https://github.com/input-output-hk/cardano-node/releases/)
```

Comment:
To generate keys for the Cardano testnet you need the `cardano-cli` [binary](https://github.com/input-output-hk/cardano-node/releases/)